elliottlib/cli/change_state_cli.py

```diff
@@ -91,18 +71,6 @@ def change_state_cli(runtime, state, advisory, default_advisories, default_advis
             if e.errata_state == state:
                 green_prefix(f"No Change ({advisory}): ")
                 click.echo(f"Target state is same as current state: {state}")
-            # we have 5 different states we can only change the state if it's in NEW_FILES or QE
```

Comment:
Removing this so that people on other teams can use elliott
to change advisory states - Like bhills from EXD team